### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Yshah112**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# E-Commerce-Project
\ No newline at end of file
+# Dash Cloths 
+Dash cloths is a online cloths website where you can puchase  attractive cloths for men and woman 
+
+# Table of contents
```
Comment (palakparmartntra) [inline README.md:4]: Add technology and directory structure. @Yshah112 @rinkishivani

File: e_commerce_project/e_commerce_app/views.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+
+# Create your views here.(
+
+def main(request):
```
Comment (palakparmartntra) [inline e_commerce_project/e_commerce_app/views.py]: Could you let me know why you created this APP? @Yshah112 @rinkishivani 
For set up, you don't need to create any app. By then, according to the feature, we need to create an app related to the feature name.

File: e_commerce_project/e_commerce_project/settings.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+]
+
+WSGI_APPLICATION = 'e_commerce_project.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/5.0/ref/settings/#databases
+
+DATABASES = {
+    'default': {
```
Comment (palakparmartntra) [inline e_commerce_project/e_commerce_project/settings.py]: Add Postgres configuration, for this project we use postgres. @Yshah112 @rinkishivani

File: e_commerce_project/e_commerce_project/urls.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path,include
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('', include('e_commerce_app.urls'))
```
Comment (palakparmartntra) [inline e_commerce_project/e_commerce_project/urls.py]: Remove this app @Yshah112 @rinkishivani

**Review by palakparmartntra — COMMENTED** [review]
Don't create another folder into repo. and there is no need to add whole template.

File: README.md [hunk 5]
```diff
@@ -1 +1,41 @@
...
+1. Clone the git repo
+
+```bash
+ django-admin startproject Dash Clothes 
+ ```
+ 
+
+
+
+>>>>>>> e26e5b04de21f28f1489051cd619608c133988fc
```
Comment (palakparmartntra) [inline README.md]: remove this. @Yshah112 @rinkishivani

File: README.md (hunk 1 above)
Comment (Yshah112) [reply README.md:4]: added technology and directory structure.

File: e_commerce_project/e_commerce_app/views.py (hunk 2 above)
Comment (Yshah112) [reply e_commerce_project/e_commerce_app/views.py]: removed application  from project directory

File: e_commerce_project/e_commerce_project/settings.py (hunk 3 above)
Comment (Yshah112) [reply e_commerce_project/e_commerce_project/settings.py]: configured Postgres db

File: e_commerce_project/e_commerce_project/urls.py (hunk 4 above)
Comment (Yshah112) [reply e_commerce_project/e_commerce_project/urls.py]: removed application url

File: README.md (hunk 5 above)
Comment (rinkishivani) [reply README.md]: Removed

**Review by palakparmartntra — APPROVED** [review]

**Pull request merged**